dss/storage/bundles.py

```diff
@@ -119,3 +120,41 @@ def _latest_version_from_object_names(object_names: typing.Iterator[str]) -> str
             version = current_version
 
     return version
+
+
+def list_available_uuids(replica: str = None, prefix: str = None, per_page: int = PerPageBounds.per_page_max,
+                         search_after: str = None, token: str = None):
+    """Return a list of available uuid's in a given prefix, removes tombstones"""
+    kwargs = dict(bucket=Replica[replica].bucket, prefix=prefix, k_page_max=per_page)
+    if search_after:
+        kwargs['start_after_key'] = search_after
+    if token:
+        kwargs['token'] = token
+
+    storage_handler = Config.get_blobstore_handle(Replica[replica])
+    prefix_iterator = storage_handler.list_v2(**kwargs)
+    keys = dict()  # type: dict
+    uuid_list = list()
+    total_keys = 0
+
+    for key, meta in prefix_iterator:
+        uuid, version = key.split('.', 1)
+        if not version.endswith(TOMBSTONE_SUFFIX):
+            search_after = key
+            if keys.get(uuid) is None:
```

Comment:
keys.setdefault(uuid, []).append(version)

Reply:
yikes, i think my merge got messed up......... this file was not supposed to be committed to change :(